rules/REQUEST-903.9001-DRUPAL-EXCLUSION-RULES.conf

```diff
@@ -310,4 +310,52 @@ SecAction "id:9001380,\
 	ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES_NAMES,\
 	ctl:ruleRemoveTargetById=942450;REQUEST_COOKIES" 
 
+
+#
+# [ WYSIWYG/CKEditor assets and upload ]
+# 
+# Disable the unnecessary requestBodyAccess and for binary uploads
+#
+# 
+#
+SecRule REQUEST_METHOD "^POST$" \
+        "id:'9001390',\
+        phase:1,\
+        t:none,\
+        pass,\
+        nolog,\
+        noauditlog,\
+        chain"
+                SecRule REQUEST_FILENAME "/admin/content/assets/add/[a-z]+$" chain
+                SecRule REQUEST_COOKIES:/S?SESS[a-f0-9]+/ "^[a-zA-Z0-9_-]+" \
+                ctl:requestBodyAccess=Off,ctl:ruleRemoveById=200004
+
+SecRule REQUEST_METHOD "^POST$" \
+        "id:'9001400',\
+        phase:1,\
+        t:none,\
+        pass,\
+        nolog,\
+        noauditlog,\
+        chain"
+                SecRule REQUEST_FILENAME "/admin/content/assets/manage/[0-9]+$" chain
+                SecRule ARGS:destination "@eq admin/content/assets" chain
+                SecRule REQUEST_HEADERS:Content-Length "@gt 31486341" chain
```

Comment:
if a multipart/form-data post exceed 30mb the rule will work.
i did it because it is not nice to get modsec log file with 700mb because an upload failed